src/Domain/Creators/BinaryCompactObjectHelpers.hpp

```diff
@@ -289,7 +301,9 @@ struct TimeDependentMapOptions {
    */
   std::unordered_map<std::string,
                      std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>
-  create_functions_of_time(const std::unordered_map<std::string, double>&
+  create_functions_of_time(const std::optional<double>& inner_radius_A,
+                           const std::optional<double>& inner_radius_B,
```

Comment:
The information isn't stored AFAICT.

Reply:
Should we store it? I think I'd prefer that over changing this function signature since right now the signature of `create_functions_of_time` is the same as `functions_of_time` that the domain creators have

Reply:
Ok pushed a fixup